samples/dashboard/demos/climate/demo.css

```diff
@@ -1,5 +1,73 @@
 @import "https://code.highcharts.com/css/highcharts.css";
 
+/* *
+ *
+ *  Colors
+ *
+ * */
+
+#time-range-selector {
+    --color-axis: #646479;
+    --color-background: #fafafa;
+    --color-grid: #fafafb;
+    --color-handle: #5352be;
+    --color-handle-line: #dadae2;
+    --color-handle-mask: #41378233;
+    --color-scroll: #9295ba;
+    --color-series: #5352be;
+}
+
+.highcharts-dashboard.dark-mode #time-range-selector {
+    --color-axis: #e7e7ec;
+    --color-background: #0d0d0d;
+    --color-grid: #838393;
+    --color-handle: #5352be;
+    --color-handle-line: #838394;
+    --color-handle-mask: #dddbef66;
+    --color-scroll: #4a4a53;
+    --color-series: #5352be;
+}
+
+#world-map {
+    --color-background: #5352be;
+    --color-map: #c7d3e9;
```

Comment:
I change the map color to #dddbef based on the feedback that the blue color was too similar to the blue markers. This new color is light purple.